internal/pkg/controllers/selinuxpolicy/selinuxpolicy_controller.go

```diff
@@ -161,63 +166,96 @@ func (r *ReconcileSP) removeFinalizer(sp *spov1alpha1.SelinuxPolicy) (reconcile.
 	return reconcile.Result{}, nil
 }
 
-func (r *ReconcileSP) reconcileConfigMap(sp *spov1alpha1.SelinuxPolicy, l logr.Logger) (reconcile.Result, error) {
-	// Define a new ConfigMap object
-	cm := r.newConfigMapForPolicy(sp)
-
-	// Check if this cm already exists
-	foundCM := &corev1.ConfigMap{}
-	err := r.client.Get(context.TODO(), types.NamespacedName{Name: cm.Name, Namespace: cm.Namespace}, foundCM)
-	if err != nil && kerrors.IsNotFound(err) {
-		l.Info("Creating a new ConfigMap", "ConfigMap.Namespace", cm.Namespace, "ConfigMap.Name", cm.Name)
-		if err = r.client.Create(context.TODO(), cm); err != nil {
-			return reconcile.Result{}, IgnoreAlreadyExists(err)
+func (r *ReconcileSP) reconcilePolicy(sp *spov1alpha1.SelinuxPolicy, l logr.Logger) (reconcile.Result, error) {
```

Comment:
The reconciliation isn't reporting any k8s events. k8s events help with troubleshooting for the operators compared to logs. It also helps e2e tests to validate the condition.

Could we add events for the invalid and valid conditions?

Reply:
This PR is already quite big. Mind if we file a Github issue and do that in a subsequent PR?

Reply:
I added a todo note to the opening comment and I'll open all the tickets at the same time

Reply:
That works me. Thanks 👍